packages/stencil/src/components/datacom-breadcrumb/usage.md

```diff
@@ -0,0 +1,16 @@
+# Breadcrumb Usage
+## Web component
+Breadcrumb has one component 'datacom-breadcrumb'
+
+```html 
+   <div>
```

Comment:
Please update these usage examples with all the latest changes you've done with the component.